samples/Commerce/WinUI/Commerce.Wasm/LinkerConfig.xml

```diff
@@ -1,7 +1,35 @@
 <linker>
   <assembly fullname="Commerce.Wasm" />
-  <assembly fullname="Uno.UI" />
-  <assembly fullname="Microsoft.Extensions.Options" />
+ 	<assembly fullname="Uno.WinUI" />
+	<assembly fullname="Uno.Extensions.Core" />
+	<assembly fullname="Uno.Extensions.Configuration" />
+	<assembly fullname="Uno.Extensions.Hosting" />
+	<assembly fullname="Uno.Extensions.Hosting.WinUI.Wasm" />
+	<assembly fullname="Uno.Extensions.Http" />
+	<!--<assembly fullname="Uno.Extensions.Http.Refit" />-->
+	<assembly fullname="Uno.Extensions.Localization" />
+	<assembly fullname="Uno.Extensions.Logging" />
+	<assembly fullname="Uno.Extensions.Logging.Serilog" />
+	<assembly fullname="Uno.Extensions.Logging.WinUI.Wasm" />
+	<assembly fullname="Uno.Extensions.Navigation" />
+	<assembly fullname="Uno.Extensions.Navigation.Toolkit.WinUI" />
+	<assembly fullname="Uno.Extensions.Navigation.WinUI" />
+	<assembly fullname="Uno.Extensions.Serialization" />
+	<assembly fullname="Uno.Extensions.Reactive" />
+	<assembly fullname="Uno.Extensions.Reactive.WinUI" />
+	<assembly fullname="Uno.Extensions.Serialization" />
+	<assembly fullname="Uno.Extensions.Serialization.Http" />
+	<assembly fullname="Uno.Extensions.Serialization.Refit" />
+	<assembly fullname="Serilog.Sinks.Console" />
+	<assembly fullname="Serilog.Sinks.Debug" />
+	<assembly fullname="Serilog.Sinks.File" />
+	<assembly fullname="Microsoft.Extensions.Options" />
+	<assembly fullname="Microsoft.Extensions.Hosting" />
+	<assembly fullname="Microsoft.Extensions.Logging" />
+	<assembly fullname="Microsoft.Extensions.Logging.Configuration" />
+	<assembly fullname="Microsoft.Extensions.Logging.Console" />
+	<assembly fullname="Microsoft.Extensions.Logging.Debug" />
+	<assembly fullname="Microsoft.Extensions.Logging.EventSource" />
```

Comment:
we're going to have to be finer grained that this, otherwise the AOT build is going to be very large.

Reply:
yep, the priority was to get it running. we should be able to pair this back a bit to work out what's essential.